onetl/strategy/incremental_strategy.py

```diff
@@ -374,6 +374,12 @@ class IncrementalBatchStrategy(OffsetMixin, BatchHWMStrategy):
 
         All of that allows to resume reading process from the *last successful batch*.
 
+    .. warning::
+
+        Not every `DB connection <https://onetl.readthedocs.io/en/0.8.1/db_connection/index.html>`_
```

Comment:
```suggestion
        Not every :ref:`DB connection <db-connections>`
```
Такие ссылки rST сам будет формировать во время генерации документации. Вести она будет сюда (без префикса `_`):

https://github.com/MobileTeleSystems/onetl/blob/develop/docs/connection/db_connection/index.rst?plain=1#L1

Текущий вариант ведет на конкретную версию документации, и ее придется обновлять при выпуске нового релиза. Это лишняя головная боль.